src/velocity-control.ts

```diff
@@ -64,4 +72,44 @@ export class VelocityControlApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take a set of optional arguments, send that to Marqeta, and
+   * have a list of Velocity Controls returned to the caller.
+   */
+  async list(search: {
+    cardProduct?: string,
+    count?: number,
+    startIndex?: number,
+    fields?: string[],
+    sortBy?: string,
+  }): Promise<{
+    success: boolean,
+    body?: VelocityControlList,
+    error?: MarqetaError,
+  }> {
+    const {
+      cardProduct = '',
+      count = 100,
+      startIndex = 0,
+      fields = '',
+      sortBy = 'lastModifiedTime',
+    } = search
```

Comment:
Here again... if empty strings will be removed by the `fire()` call... then why set `cardProduct` and `fields` to have default values of `''` - when there's no difference if you have:

```typescript
     const {
       cardProduct,
       count = 100,
       startIndex = 0,
       fields,
       sortBy = 'lastModifiedTime',
     } = search
```

and not have the defaults that will be automatically removed. Right?